src/tests/FlashOWare.Tool.Cli.Tests/FlashOWare.Tool.Cli.Tests.csproj

```diff
@@ -22,7 +22,7 @@
   <ItemGroup>
     <PackageReference Include="DiffPlex" Version="1.7.1" />
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic" Version="4.8.0" />
-    <PackageReference Include="NuGet.Packaging" Version="6.8.0" />
+    <PackageReference Include="NuGet.Packaging" Version="[6.*,6.12.0)" />
```

Comment:
Unless we are also publishing tests, the version `6.*` will be enough as it will resolve to the highest acceptable stable version.

When the package will be provided for .NET 9 with version `6.13.0`, then it will already be compatible. If they start using `7.x` then there could also be a section specifically for .NET 9 or the upper limit (as shown by your change) of the version can be increased.

Reply:
I see your point.

Since you mentioned "section", I was playing with this idea:
```xml
<PackageReference Include="NuGet.Packaging" Version="[6.*,6.4.0)" Condition="'$(TargetFramework)' == 'net6.0'" />
<PackageReference Include="NuGet.Packaging" Version="[6.*,6.8.0)" Condition="'$(TargetFramework)' == 'net7.0'" />
<PackageReference Include="NuGet.Packaging" Version="[6.*,6.12.0)" Condition="'$(TargetFramework)' == 'net8.0'" />
```
The older versions are still serviced and have the latest vulnerabilities fixed.

But the more I think about it, the more I get convinced by your suggestion.

Reply:
Changed to
```diff
-  <PackageReference Include="NuGet.Packaging" Version="6.8.0" />
+  <PackageReference Include="NuGet.Packaging" Version="6.*" />
 </ItemGroup>
```